golang/simsimd.go

```diff
@@ -16,12 +16,12 @@ inline static simsimd_f32_t sqeuclidean_f32(simsimd_f32_t const* a, simsimd_f32_
 import "C"
 
 
-// CosineI8 computes the cosine similarity between two i8 vectors using the most suitable SIMD instruction set available.
+// CosineI8 computes the cosine similarity distance between two i8 vectors using the most suitable SIMD instruction set available.
```

Comment:
The "Cosine Distance" and "Cosine Similarity" differ. It shouldn't be "similarity distance", only one of those should be kept 🤗